src/modules/claims/claims.service.ts

```diff
@@ -757,7 +757,20 @@ export class ClaimsService {
     registrationTypes = [RegistrationTypes.OffChain],
     claim,
   }: PublishPublicClaimOptions): Promise<string | undefined> {
-    claim.token = claim.token || token;
+    claim.token = claim.token || (token as string);
```

Comment:
More safer to throw error if neither `claim.token` nor `token` provided

Reply:
@JGiter is this what you were thinking?